### ISSUE TITLE
New icons from reddit

### ISSUE THREAD

**Issue by skmp** [reicast/reicast-emulator#829]
![reicast-symbiotics_classic symbiotics](https://cloud.githubusercontent.com/assets/393266/9783838/8a6ec614-57a7-11e5-9f9b-a2c47bf0437c.jpg)

Icon PNGs and psds in [gdrive](https://drive.google.com/open?id=0B6gSdFz3bTVyZG40NEM4MFRQdTA)

**Comment by ohnozen** [comment]
Hi everyone, I'm the guy that designed this icon set! I'm looking forward to hearing about what everyone thinks about my icon design. I'm open to any criticism and suggestions, even if you just want to tell me I suck and smell really bad :)

I haven't worked on this set in a few months, so I'm sure there's a few things here and there I could tweak and adjust (I've already seen about four things since typing this). Plus I need to work on the icon and symbols for the in-game menu.

**Comment by LoungeKatt** [comment]
This will require a pretty hefty layout rebuild to look more "material" but they look slick

**Comment by Holzhaus** [comment]
Really nice. These are raster images, not vector ones?

Also, CHD images could deserve their own icons (maybe in green color?).

Awesome work btw.

**Comment by ohnozen** [comment]
The .zip file contains both .png and .svg files.

I originally had a "Reicast Purple" in the main palette. I suppose I could re-introduce it and that color can be used for CHD files?

And thanks for the compliments! I originally started this as "do some nice icon work," which quickly turned into, "let's redesign everything from the ground up to look unified and amazing and [unintelligible graphic designer ranting]." Currently my other designs and visual ideas are spread across three or four Illustrator files. Hopefully when I get home tonight I'll have to really sit down and polish some of the extra ideas and compile a more updated visual system.

But for now here's some examples of some extremely early logo work I did, showcases the purple I sampled from the Reicast logo:

![earliest-logo-concepts](https://cloud.githubusercontent.com/assets/14218478/9799762/f0400558-57ce-11e5-8efc-68879550d944.jpg)

And I was also working on other visual ideas, like a pop-up icon when the VMU beeps (instead of having it onscreen all the time or having to play an audible "beeeep"):

![vmu-beep-demo-01](https://cloud.githubusercontent.com/assets/14218478/9799818/2d5660ae-57cf-11e5-8214-4e45ca1f0b27.jpg)
![vmu-beep-demo-02](https://cloud.githubusercontent.com/assets/14218478/9799817/2d540818-57cf-11e5-84a1-c29e596fedbf.jpg)

**Comment by skmp** [comment]
(I'm a bit on withdrawal phase, but I must say I really love the logo
ideas. Will comment as soon as I get better)

On Thu, Sep 10, 2015 at 10:19 PM, ohnozen <notifications@github.com> wrote:

> The .zip file contains both .png and .svg files.
>
> I originally had a "Reicast Purple" in the main palette. I suppose I could
> re-introduce it and that color can be used for CHD files?
>
> And thanks for the compliments! I originally started this as "do some nice
> icon work," which quickly turned into, "let's redesign everything from the
> ground up to look unified and amazing and [unintelligible graphic designer
> ranting]." Currently my other designs and visual ideas are spread across
> three or four Illustrator files. Hopefully when I get home tonight I'll
> have to really sit down and polish some of the extra ideas and compile a
> more updated visual system.
>
> But for now here's some examples of some extremely early logo work I did,
> showcases the purple I sampled from the Reicast logo:
>
> [image: earliest-logo-concepts]
> <https://cloud.githubusercontent.com/assets/14218478/9799762/f0400558-57ce-11e5-8efc-68879550d944.jpg>
>
> And I was also working on other visual ideas, like a pop-up icon when the
> VMU beeps (instead of having it onscreen all the time or having to play an
> audible "beeeep"):
>
> [image: vmu-beep-demo-01]
> <https://cloud.githubusercontent.com/assets/14218478/9799818/2d5660ae-57cf-11e5-8214-4e45ca1f0b27.jpg>
> [image: vmu-beep-demo-02]
> <https://cloud.githubusercontent.com/assets/14218478/9799817/2d540818-57cf-11e5-84a1-c29e596fedbf.jpg>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/reicast/reicast-emulator/issues/829#issuecomment-139368107>
> .
>



-- 
~skmp

**Comment by LoungeKatt** [comment]
Agreed. I like the first row best. The others are a bit overwhelming.

**Comment by skmp** [comment]
Okay, I'm back into active status. I think I like the icons on the first row as well. We need to pick something and stick to it though, that's how branding is built (says the guy that has changed names and logos on this project many times)

**Comment by AbandonedCart** [comment]
Was this forgotten?
